package.json

```diff
@@ -29,7 +29,7 @@
   "types": "./typings/lib/msRestAzure.d.ts",
   "license": "MIT",
   "dependencies": {
-    "ms-rest-js": "^0.3.0"
+    "ms-rest-js": "RikkiGibson/ms-rest-js#HttpClient"
```

Comment:
ahh, actually no longer sure what branch to base this work on, since the version of ms-rest-js has to be from Azure/master. What do you think @daschult?